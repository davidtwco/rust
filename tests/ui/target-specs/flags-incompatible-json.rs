//@ compile-flags: -Ztarget-spec=aarch="aarch64" --target=foo.json
//@ error-pattern: JSON target specifications are incompatible with `-Ztarget-spec`
//@ needs-llvm-components:
