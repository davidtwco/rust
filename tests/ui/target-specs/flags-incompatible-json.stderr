error: JSON target specifications are incompatible with `-Ztarget-spec`

